e2e/cypress/plugins/typescriptPreprocessor.js

```diff
@@ -1,32 +1,18 @@
 const wp = require('@cypress/webpack-preprocessor');
-const { resolve } = require('path');
-
-const anyNodeModules = /node_modules/;
-const packageRoot = resolve(`${__dirname}/../../`);
-const packageModules = `${packageRoot}/node_modules`;
 
 const webpackOptions = {
   module: {
     rules: [
       {
-        include: (modulePath) => {
-          if (!anyNodeModules.test(modulePath)) {
-            // Is a file within the project
-            return true;
-          } else {
-            // Is a file within this package
-            return modulePath.startsWith(packageRoot) && !modulePath.startsWith(packageModules);
-          }
-        },
         test: /\.ts$/,
-        use: [
-          {
-            loader: 'ts-loader',
-            options: {
-              transpileOnly: true,
-            },
+        exclude: /node_modules/,
+        use: {
+          loader: 'esbuild-loader',
+          options: {
+            target: 'es2020',
+            format: undefined,
           },
-        ],
+        },
```

Comment:
Do we need to apply same changes to [/packages/grafana-e2e/cypress/plugins/](https://github.com/grafana/grafana/blob/main/packages/grafana-e2e/cypress/plugins/typescriptPreprocessor.js)?

Reply:
We could do similar changes here but I'm a little hesitant in case it breaks tests for some plugins.